src/main/resources/com/rsmart/kuali/kfs/cr/ojb-cr.xml

```diff
@@ -33,6 +33,7 @@
     	<field-descriptor name="bankCode" column="BNK_CD" jdbc-type="VARCHAR" />
     	<field-descriptor name="cancelDocHdrId" column="CNCL_DOC_HDR_ID" jdbc-type="VARCHAR" />
     	<field-descriptor name="versionNumber" locking="true" column="VER_NBR" jdbc-type="BIGINT" />
+    	<field-descriptor name="active" column="ACTV_IND" jdbc-type="VARCHAR" conversion="org.kuali.kfs.core.framework.persistence.ojb.conversion.OjbCharBooleanConversion" />

```

Comment:
Are these changes part of KFSPTS-30132? I don't think this file should be in this PR.